File: src/main/java/club/libridge/libridgebackend/core/GameConstants.java
package club.libridge.libridgebackend.core;

public final class GameConstants {

    private GameConstants() {
        throw new IllegalStateException("Utility class");
    }

    public static final int NUMBER_OF_HANDS = 4;
    public static final int SIZE_OF_HAND = 13;
    public static final int NUMBER_OF_TRICKS_IN_A_COMPLETE_HAND = SIZE_OF_HAND;
    public static final int COMPLETE_TRICK_NUMBER_OF_CARDS = NUMBER_OF_HANDS;

}

File: src/main/scala/scalabridge/GameConstants.scala
package scalabridge

object GameConstants {
  def instance: GameConstants.type = this
  def NUMBER_OF_HANDS: Int = 4
  def SIZE_OF_HAND: Int = 13
  def NUMBER_OF_TRICKS_IN_A_COMPLETE_HAND: Int = SIZE_OF_HAND
  def COMPLETE_TRICK_NUMBER_OF_CARDS: Int = NUMBER_OF_HANDS
}
